pyartcd/pyartcd/pipelines/operator_sdk_sync.py

```diff
@@ -0,0 +1,137 @@
+import os
+import re
+import subprocess
+
+import click
+import koji
+import yaml
+from errata_tool import Erratum
+from pyartcd import constants, util
+from pyartcd.cli import cli, click_coroutine, pass_runtime
+from pyartcd.runtime import Runtime
+
+
+class OperatorSDKPipeline:
+    def __init__(self, runtime: Runtime, group: str, assembly: str, nvr: str, prerelease: bool, updatelatest: bool) -> None:
+        self.runtime = runtime
+        self._logger = runtime.logger
+        self.version = group.split("-")[1]
+        self.assembly = assembly
+        self.prerelease = prerelease
+        self.nvr = nvr
+        self.updatelatest = updatelatest
+        self.sdk = "operator-sdk"
+        self.group = group
+        self.extra_ad_id = ""
+        self.parent_jira_key = ""
+        self._jira_client = runtime.new_jira_client()
+
+    async def run(self):
+        if self.assembly:
+            group_config = await util.load_group_config(self.group, self.assembly, env=os.environ.copy())
+            self.extra_ad_id = group_config.get("advisories", {}).get("extras", 0)
+            self.parent_jira_key = group_config.get("release_jira")
+            advisory = Erratum(errata_id=self.extra_ad_id)
+            self._logger.info("Check advisory status ...")
+            if advisory.errata_state in ["QE", "NEW_FILES"]:
+                self._logger.info("Advisory status not in REL_PREP yet ...")
+                return
+            if advisory.errata_state == "SHIPPED_LIVE":
+                self._logger.info("Advisory status already in SHIPPED_LIVE, update subtask 9 ...")
+                self._update_jira(self.parent_jira_key, 8, "Advisory status already in SHIPPED_LIVE")
```

Comment:
This could cause an issue if the parent jira template changes. This could be improved by writing a keyword or label based simple search to find the sub jira.

If you don't have time to do it for now, I would suggest adding a comment to mention the caveat.